src/test/java/com/algorand/algosdk/abi/TestTypes.java

```diff
@@ -238,6 +238,22 @@ public void TestTypeFromStringValid() {
                         )
                 )
         );
+        assertThat(ABIType.valueOf("(uint32,(uint64,bool)[10],byte)")).isEqualTo(
```

Comment:
maybe an additional one that puts the static array at the end so it tests the end-of-string condition in addition to the comma condition.

Reply:
got it, I will mimic the cases in avm-abi one